pkg/plugins/policies/core/ordered/ordered.go

```diff
@@ -25,6 +25,8 @@ var Policies = []plugins.PluginName{
 	plugins.PluginName(meshtcproute_api.MeshTCPRouteResourceTypeDescriptor.KumactlArg),
 	// For other policies order isn't important at the moment
 	plugins.PluginName(meshloadbalancingstrategy_api.MeshLoadBalancingStrategyResourceTypeDescriptor.KumactlArg),
+	// has to be before MeshAccessLog so logs can access logs are added to the chains
```

Comment:
sounds a bit weird, "logs can access logs" ? shouldn't this be 

```suggestion
	// has to be before MeshAccessLog so the plugin can access log filters that are added to the filter chains
```

?